src/components/booking/booking-card/BookingCard.vue

```diff
@@ -0,0 +1,333 @@
+
+          <div class="row mb-5">
+            <h3 class="section-title txt-2 semibold">
+              Trip details
+            </h3>
+          </div>
+
+          <div class="row mb-5">
+            <section class="section">
+              <p class="title txt-5 light">Arrival date</p>
+              <p class="value txt-5 semibold">{{ startDate }}</p>
+            </section>
+            <section class="section">
+              <p class="title txt-5 light">Departure date</p>
+              <p class="value txt-5 semibold">{{ endDate }}</p>
+            </section>
+          </div>
+
+          <div class="row">
+            <section class="section">
+              <p class="title txt-5 light">Days</p>
+              <p class="value txt-5 semibold">{{ duration }}</p>
+            </section>
+            <section class="section">
+              <p class="title txt-5 light">Guests</p>
+              <p class="value txt-5 semibold">{{ guests }}</p>
+            </section>
+          </div>
+
+        </div>
+
+        <div class="customer-info col">
+
+          <div class="row mb-5">
+            <h3 class="section-title txt-2 semibold">
+              Booked by
+            </h3>
+          </div>
+
+          <div class="row mb-5">
+            <section class="section">
+              <p class="title txt-5 light">Name</p>
+              <p class="value txt-5 semibold">{{ customerFullName }}</p>
+            </section>
+          </div>
+
+          <div class="row">
+            <section class="section">
+              <p class="title txt-5 light">Email</p>
+              <p class="value txt-5 semibold">{{ customerEmail }}</p>
+            </section>
+          </div>
+
+        </div>
+
+        <div class="payment-info col">
+
+          <div class="row mb-5">
+            <h3 class="section-title txt-2 semibold">
+              Payments
+            </h3>
+          </div>
+
+          <div class="row payment">
+            <p class="label txt-5">{{ calculationPricePerGuest }}</p>
+            <p class="value txt-5">{{ price }}</p>
+          </div>
+
+          <div class="row payment">
+            <p class="label txt-5">Deposit</p>
+            <p class="value txt-5">{{ depositPrice }}</p>
+          </div>
+
+          <div class="row payment">
+            <p class="label txt-5">Due amount</p>
+            <p class="value txt-5">{{ dueAmount }}</p>
+          </div>
+
+          <div class="row payment">
+            <p class="label txt-5 semibold">Total</p>
+            <p class="value txt-5 semibold">{{ price }}</p>
+          </div>
+
+        </div>
+
+      </div>
+
+    </div>
+  </div>
+</template>
+
+<script>
+import { IMG } from '@fishtripr/constants'
+import { responsiveHandler } from '../../../mixins/responsiveHandler'
+import Avatar from '../../avatar/Avatar'
+
+export default {
+  name: 'booking-card',
+  mixins: [ responsiveHandler ],
+  components: { Avatar },
+  props: {
+    startDate: { type: String },
```

Comment:
Any reasons you pass around strings instead of dates? It would seem more logical to me to have this as a date and have the display logic to convert it to a string inside the component instead of outside of it?

Reply:
Because I don't want to make the UI take the responsability of the format, if tomorrow we decide to change date format, we should not touch the UI IMO

Reply:
I actually think that the date format is absolutely a UI concern. The code should in almost all places not be aware of the format, except if it gets displayed to a user.

Reply:
It could be great to have a deep conversation about that because of some doubt
I will make a ticket in backlog to do that at some point